lib/customs.js

```diff
@@ -65,10 +65,7 @@ module.exports = function (log, error) {
       handleCustomsResult.bind(request),
       err => {
         log.error({ op: 'customs.check.1', email: email, action: action, err: err })
-        // If this happens, either:
```

Comment:
I wonder if it makes sense here to handle these errors differently in the 1, 2 scenarios. 

For scenario 1, if server isn't pointing to a real custom server, maybe we fail open. This would hopefully keep us compatible with self hosters and custom configs since it is backwards compatible.

For scenario 2, my primary concern with failing closed is that we could DOS users with a faulty custom server. In practice, I am not sure how often (if ever) we return internal errors from custom server, so this could be a non-issue.

Reply:
> For scenario 2, my primary concern with failing closed is that we could DoS
> users with a faulty custom server.

I suspect this may actually be the desired behavior - we mess up our customs-server config, cause a production outage, get notified promptly and fix it ASAP.

Reply:
> I suspect this may actually be the desired behavior - we mess up our customs-server config, cause a production outage, get notified promptly and fix it ASAP.

I agree with this rationale, it seems strange to me that we'd make affordances for the customs server but none for any others. If there's a problem with the auth-server that prevents users from signing in, we'd roll back or fix it ASAP whereas in this scenario we are saying if there's a problem with the customs server, we'll just ignore it.